api/repo.go

```diff
@@ -427,6 +427,11 @@ func GetRepos(c *gin.Context) {
 //   type: integer
 //   maximum: 100
 //   default: 10
+// - in: query
+//   name: sort_by
+//   description: How to sort the results
+//   type: string
+//   default: name
```

Comment:
should we add `enum` here? it can really only be `name` or `latest` right?

see example here: https://github.com/go-vela/server/blob/61358d2df68207f980a7845270193f50e0eb5709/api/secret.go#L43-L46